config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-discovery.yaml

```diff
@@ -41,6 +41,46 @@ periodics:
           memory: "9000Mi"
           cpu: 2000m
 
+- name: ci-kubernetes-e2e-kubeadm-kinder-discovery-1-19
+  interval: 12h
+  decorate: true
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-kubeadm
+    testgrid-tab-name: kubeadm-kinder-discovery-1-19
+    testgrid-alert-email: kubernetes-sig-cluster-lifecycle+testgrid@googlegroups.com, kubernetes-release-team@googlegroups.com
```

Comment:
no need for the release-team email here.
it's only needed for the x-on-y, upgrade and regular jobs.

[1]

Reply:
removed.